### PR TITLE
docs: Add docs and rename plotting to chartly

### PR DESCRIPTION
**Pull request by Azendae-Popo**
# 🥇 Generic Pull Request 🥇
### Purpose & Objective

***

- [x] This PR satisfies the PR checklist.

### CODE REVIEW

**Review by cmbengue-ec-intl — CHANGES_REQUESTED** [review]
Please update based on our discussion.

**Comment by Azendae-Popo** [comment]
image

**Review by cmbengue-ec-intl — CHANGES_REQUESTED** [review]
Minor refactoring is needed. See the comments below.

File: src/chartly/chartly.py
[the remark this answers is not in the document]
Comment (Azendae-Popo) [reply src/chartly/chartly.py:1]: Alrighty

**Review by cmbengue-ec-intl — APPROVED** [review]
This looks good. Great work on this. 💯

**Pull request merged**